libraries/rib-base/src/test/java/com/badoo/ribs/routing/state/RoutingStatePoolTest.kt

```diff
@@ -633,6 +633,17 @@ class RoutingStatePoolTest {
     }
     // endregion
 
+    @Test
+    fun `On Deactivate, Node routing is removed from pendingDeactivate`() {
+        createEmptyPool()
+        pool.accept(Transaction.from(
+            Add(helperContentViewParented1.routing),
+            Deactivate(helperContentViewParented1.routing)
+        ))
+
+        assertEquals(true, pool.state.pendingDeactivate.isEmpty())
+    }
```

Comment:
This doesn't seem to prove that it's removed correctly only that it's not there in the end – if it was never there in the first place, then this test passes just the same.

Reply:
Added one more test that verifies, that `pool.state.pendingDeactivate` contains `helperContentViewParented1.routing` during `DeactivateAction`